pages/crash_report_page.py

```diff
@@ -29,14 +29,15 @@ def click_reports_tab(self):
         self.selenium.find_element(*self._report_tab_button_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: not self.is_element_visible(None, *self._reports_loading_locator))
 
-    class Report(CrashStatsBasePage):
+    class Report(object):
 
         _product_locator = (By.CSS_SELECTOR, 'td:nth-of-type(3)')
         _version_locator = (By.CSS_SELECTOR, 'td:nth-of-type(4)')
         _report_date_link_locator = (By.CSS_SELECTOR, '#reports-list .report-date_processed > a')
 
-        def __init__(self, testsetup, element):
-            CrashStatsBasePage.__init__(self, testsetup)
+        def __init__(self, base_url, selenium, element):
```

Comment:
If you base this class on `PageRegion` then you won't need this `__init__` method at all. It looks like you'll also need to add `base_url` to your `PageRegion`, as I've done in oneanddone-tests [1].

Of course I may be misunderstanding how all of this works, so if I am please feel free to correct me.

[1] https://github.com/mozilla/oneanddone-tests/blob/master/pages/page.py#L55